roda-common/roda-common-data/src/main/java/org/roda/core/data/common/RodaConstants.java

```diff
@@ -328,6 +329,7 @@
   public static final String API_ACKNOWLEDGE = "acknowledge";
   public static final String API_STOP = "stop";
   public static final String API_REPORTS = "reports";
+  public static final String API_SYNCRONIZATION = "synchronization";
```

Comment:
This is a path parameter, so change the prefix to API_PATH_PARAM and move it to the place where the path parameters constants are. (line ~ 390)